src/css/skills.scss

```diff
@@ -0,0 +1,130 @@
+@import './core/constants';
+@import './core/mixins';
+
+#skills {
+  padding: 35px 70px;
+  min-height: 600px;
+  background-image: url('https://static.wixstatic.com/media/05e3dc_cc646f530d6846828d53dd3216ec8023~mv2.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+
+  .content {
+    width: 950px;
+    margin-top: 100px;
+    h2 {
+      color: $white;
+      font-size: $font-xl;
+    }
+    hr {
+      background: $white;
+    }
+
+    .skill-board {
+      @include size(100%, 435px);
+      .light {
+        background-color: $skill-bg-light;
+        .skill-title {
+          color: $black;
+        }
+        .skill-level {
+          color: $red;
+          font-weight: 600;
+        }
+      }
+      .dark {
+        background-color: $skill-bg-dark;
+        .skill-title {
+          color: $black;
+        }
+        .skill-level {
+          color: $navy;
+          font-weight: 600;
+        }
+      }
+
+      li {
+        float: left;
+        .skill-title,
+        .skill-level {
+          text-align: center;
+        }
+
+        &.small {
+          @include size(145px);
+          .skill-title {
+            position: relative;
+            top: 50%;
+            font-size: 8px;
```

Comment:
What you are doing is okey, but maybe try something like https://drafts.csswg.org/css-values/#viewport-relative-lengths